biogeochem/FatesSoilBGCFluxMod.F90

```diff
@@ -997,9 +997,9 @@ subroutine FluxIntoLitterPools(csite, bc_in, bc_out)
        
        select case (element_list(el))
        case (carbon12_element)
-          bc_out%litt_flux_cel_c_si(:) = 0._r8
-          bc_out%litt_flux_lig_c_si(:) = 0._r8
-          bc_out%litt_flux_lab_c_si(:) = 0._r8
+          bc_out%litt_flux_cel_c_si(:) = 0.0_r8
+          bc_out%litt_flux_lig_c_si(:) = 0.0_r8
+          bc_out%litt_flux_lab_c_si(:) = 0.0_r8
```

Comment:
Embarrassing question I've never got my head around. Does this matter? Like, are there circumstances where you get a different answer with a 0.0_r8 than a 0._r8?

Reply:
nope, I don't think it matters.  This was just an artifact too, I didn't make the change for stylistic reasons.  This is what happened.  I was testing the code by forcing a non zero decomposition flux into E3SM with a value of 1.0. I wanted to see if E3SM was closing its carbon budget with a constant and known input flux from FATES. When I reverted it, I just added a zero.